### PR TITLE
[SW-1050] Remove usage of SDK v11 transaction schema

### PR DESCRIPTION
**Pull request by subhod-i**

### CODE REVIEW

**Review by pstachel — APPROVED** [review]

File: src/popup/utils/utils.ts [hunk 1]
```diff
@@ -374,7 +374,7 @@ export function categorizeContractCallTxObject(transaction: ITransaction): {
   url?: string
   note?: string
 } | null {
-  if (!compareCaseInsensitive(transaction.tx.type, SCHEMA.TX_TYPE.contractCall)) {
+  if (!compareCaseInsensitive(transaction.tx.type, Tag[Tag.ContractCallTx])) {
```
Comment (peronczyk) [inline src/popup/utils/utils.ts:377]: Maybe the `tx.type` can be now compared directly to the `Tag[Tag.ContractCallTx]`. I mean do they have the same casing?

File: src/types/index.ts [hunk 2]
```diff
@@ -367,8 +367,8 @@ export interface ITx {
   recipientId?: string
   senderId?: string
   selectedTokenContractId?: string
-  tag?: Tag;
-  type: keyof typeof Tag,
+  tag?: Tag
+  type: string
```
Comment (peronczyk) [inline src/types/index.ts:371]: TS properties should end up with `;`.

Comment (peronczyk) [inline src/types/index.ts:371]: I think this change is bad - we need to have explicit list of possible values.

**Review by peronczyk — CHANGES_REQUESTED** [review]

**Comment by CharisSiarampalis** [comment]
Lgtm

File: src/popup/utils/utils.ts (hunk 1 above)
Comment (subhod-i) [reply src/popup/utils/utils.ts:377]: As discussed, I have added code comments for the usage of tx `type`. We need to support case sensitive contract call URLs

File: src/types/index.ts (hunk 2 above)
Comment (subhod-i) [reply src/types/index.ts]: Fixed it. Added comment too.

**Review by peronczyk — APPROVED** [review]

**Pull request merged**